tools/next-version

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const { program } = require('commander')
+
+const VERSION_FILE = 'version.json'
+
+program
+  .option('-d, --debug', 'enable extreme logging')
+
+const calculateNewVersion = () => {
+  const versionFile = fs.readFileSync(VERSION_FILE)
+  // versionCode is just used in the integreat-react-native-app
+  const { versionName, versionCode } = JSON.parse(versionFile)
+  const versionNameParts = versionName.split('.').map(it => parseInt(it))
+
+  const date = new Date()
+  const year = date.getFullYear()
+  const month = (date.getMonth() + 1)
+
+  const versionNameCounter = year === versionNameParts[0] && month === versionNameParts[1] ? versionNameParts[2] + 1 : 0
```

Comment:
```suggestion
  const versionNameCounter = (year === versionNameParts[0] && month === versionNameParts[1]) ? versionNameParts[2] + 1 : 0
```

Reply:
Won't do for consistency with the webapp script for now.